src/styles/_catalog.scss

```diff
@@ -1,10 +1,20 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px;
```

Comment:
it's generally ok, yet better to replace margins with gap
currently you've added gaps on top of margins
![Image 2022-04-01 14 24 48](https://user-images.githubusercontent.com/39308717/161254382-56c807b3-9f27-4662-920e-4041a8194941.png)